src/pages/RealmScans/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+
+export const RealmScans: pageInterface = {
+  name: 'RealmScans',
+  domain: 'https://realmscans.com',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    return j.$('#screenmode').length > 0;
+  },
+  isOverviewPage(url) {
+    return utils.urlPart(url, 3) === 'series';
+  },
+  getImage() {
+    return j.$('div.thumb img').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('.ts-breadcrumb li:nth-child(2) [itemprop="name"]').text();
+    },
+    getIdentifier(url) {
+      return RealmScans.overview!.getIdentifier(RealmScans.sync.getOverviewUrl(url));
+    },
+    getOverviewUrl(url) {
+      return j.$('.ts-breadcrumb li:nth-child(2) a').attr('href') || '';
+    },
+    getEpisode(url) {
+      const episodePart = utils.urlPart(url, 3);
+
+      const temp = episodePart.match(/-chapter-(\d+)/im);
+
+      if (!temp) return NaN;
+
+      return Number(temp[1]);
+    },
+    nextEpUrl(url) {
+      return j.$('a.ch-next-btn:first').attr('src');
```

Comment:
src is for images so `.attr('href')` is correct here. And additionally it has to return undefined if there is no next chapter, but it would return `#/next/` here https://realmscans.com/one-in-seven-billion-irregulars-chapter-25/